.changeset/clean-sloths-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fix send attachments option when using rocket.chat from an iOS browser moving the option outside from composer dropdown menu.
```

Comment:
https://developer.rocket.chat/v1/docs/development-workflow#branching-and-pull-requests
```suggestion
Fixes the send attachments option not working on iOS browsers by moving it from the composer dropdown menu to the composer primary actions.
```